RetrospectiveExtension.Frontend/webpack.config.js

```diff
@@ -12,11 +12,7 @@ module.exports = {
   output: {
     path: BUILD_DIR,
     filename: './reflect-bundle.js',
-    libraryTarget: 'amd'
```

Comment:
why was this removed?

Reply:
If you include it, the bundle js file will be built and expected to be loaded in a requirejs environment. With the old initialization, the `vss.load(..)` method can handle the bundle file but in the new sdk, it'll cause the `define is not defined` error. You can have a look at the `index.html` and `index.tsx` files for the initialization process.